### ISSUE TITLE
License?

### ISSUE THREAD

**Issue by davclark** [rochelleterman/FSUtext#1]
Hey - I'm here following up on the carpentries NLP discussion... did you have any wishes regarding usage rights? I'm happy to submit a PR with your preferred license. I suppose copyright might be vague because you're using stuff from elsewhere?

**Comment by rochelleterman** [comment]
Hi! Yes, I need guidance on this. I did my best to achieve permission, and provide credit for the materials that I adapted. Ideally, I would like a license that allows others to do the same: adapt with attribution. Which license would you recommend for something like this?

**Comment by davclark** [comment]
Yeah - in this case, I think the best case is to describe the situation. I mean, for you personally, I guess there's very little risk, but ambiguous rights could inhibit use by an org like the Carpentries. I wonder how the protections for using copyrighted material for teaching apply here?

Anyway, I guess if you wanted to get copyright tight, you'd need to check back with where you got the materials. Is it worth it?

Barring that, you might just explain where you got stuff and say that *your* changes are licensed CC-BY or MIT or whatever, but that you don't hold copyright to much of it, and provide whatever sign-posts to the sources that seem worth it to you for now... once you're a fancy professor, you could also probably get the school's lawyers to provide input / assistance on your curricula.

And of course obligatory: I am not a lawyer, so you should interpret the above with as much authority as a thought-provoking dream.

> On the one hand information wants to be expensive, because it's so valuable. The right information in the right place just changes your life. On the other hand, information wants to be free, because the cost of getting it out is getting lower and lower all the time.
